<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDockWidget</class>
 <widget class="QDockWidget" name="SearchDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>11</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>14</y>
      <width>144</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="classCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>73</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Classes</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="individualCB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>93</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Individuals</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="oPropertyCB">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>70</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Object Properties</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dPropertyCB">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>100</y>
      <width>118</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Properties</string>
    </property>
   </widget>
   <widget class="QListWidget" name="resultList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sorting">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>40</y>
      <width>144</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Default</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alphabetically</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Importance</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>40</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
